corehq/apps/cloudcare/views.py

```diff
@@ -544,9 +544,10 @@ def _fail(error):
             return _fail(_("No corresponding application found in this project."))
 
     restore_as_user, set_cookie = FormplayerMain.get_restore_as_user(request, domain)
-    force_login_as = not restore_as_user.is_commcare_user()
+    force_login_as = toggles.SMART_LINKS_FOR_WEB_USERS and not restore_as_user.is_commcare_user()
```

Comment:
Shouldn't this be something like `toggles.SMART_LINKS_FOR_WEB_USERS.enabled(domain)`?

Reply:
Oh my. Yes. And I forgot the `not`.